src/main/java/org/jenkinsci/plugins/statsd/StatsdPeriodicWorker.java

```diff
@@ -28,11 +30,14 @@ protected void doRun() {
 
         StatsdConfig config = StatsdConfig.get();
 
-        Map<String,Long> queueItems = new HashMap<>();
+        List<Map.Entry<String,Long>> queueItems = new ArrayList<>();
```

Comment:
nit - This is very backwards for Java, the data structure you want is probably something like `<Map<String, List<Long>>` instead.